### ISSUE TITLE
The duration bar doesn't dynamically change

### ISSUE THREAD

**Issue by Root1-2** [ModernFlyouts-Community/ModernFlyouts#1337]
### ModernFlyouts Version

0.10.0.0

### Windows version

Windows 11 23H2

### Platform

x64

### Area(s) with issue?

Media Flyout

### Steps to reproduce

Whenever I play a song with Dopamine music player, the song timestamp stuck a 0:00 always, it always updates if I pause the song using the flyout, otherwise the timestamp doesn't update. At least, Windows Media Player update the timestamp when the flyout get hidden, but Dopamine doesn't.

![Screenshot 2024-09-08 152905](https://github.com/user-attachments/assets/489612e2-e07d-4f5b-9032-337c7b6d8a88)

### ✔️ Expected Behavior

The timestamp should update dynamically, second by second right?

### ❌ Actual Behavior

Stuck timestamp.

**Comment by manstheram** [comment]
Does it work normally with the Windows popup or is it only with the custom Flyouts?

**Comment by Root1-2** [comment]
My windows popup only shows the soundbar flyouts. Doesn't show the media control flyout.

**Comment by manstheram** [comment]
It's probably an issue with Dopamine, but the app is open-source, I think, so I'll check later if there's anything wrong there.

**Comment by Root1-2** [comment]
Yes it is an open source. Thank you. And if you need any video of the problem to troubleshoot or anything, please reach me out.

**Comment by manstheram** [comment]
I checked and saw that there are two versions of Dopamine, can you try both of them?
https://github.com/digimezzo/dopamine
https://github.com/digimezzo/dopamine-windows

**Comment by Root1-2** [comment]
So I tried the old one (Dopamine-windows) and this version don't even show up in the modern flyout when playing a song. Whereas the new one (Dopamine) made using JavaScript+TypeScript (Angular) shows up in the modern flyout when playing a song

**Comment by CaptainDeSync** [comment]
v0.9.3 does this too with Tidal. The only time it updates is when the Tidal app is interacted with by either clicking on it, opening it, pausing or playing music, etc.

**Comment by Root1-2** [comment]
> v0.9.3 does this too with Tidal. The only time it updates is when the Tidal app is interacted with by either clicking on it, opening it, pausing or playing music, etc.

Yep, happens with lots of apps such as MS Edge (if watching yt video using edge)

**Comment by manstheram** [comment]
Does it work for any apps?

**Comment by manstheram** [comment]
I did a fair bit of testing, and all that resulted in was the fact that whatever api this app uses for the playback time isn't sending a notification back to the app telling it to update. So there's 3 ways I could solve this:
1. Mess around with the COM interface stuff.
2. Use a workaround that spams trying to update the time or at least do it at regular intervals and then smooth out values - don't what's so bad about this but I have a feeling it's not the best idea.
3. Use a workaround where I just add 1 second to the playback time every second - issue with this is that the song could go out of sync or be running at a different playback rate.
